js/statustracker/static/index.svelte

```diff
@@ -47,7 +47,7 @@
 <script lang="ts">
 	import { onDestroy } from "svelte";
 	// TODO: revisit this
-	import { app_state } from "../../app/src/stores";
+
```

Comment:
I guess we can finally delete the TODO comment 😂

Reply:
I did wonder what that was actually referring to tbh, although i wrote it. Maybe it was indeed this.